tinker/static/assets/css/tinker.css

```diff
@@ -593,6 +593,11 @@ label.checkbox {
     font-size: 85%;
 }
 
+.courseleaf-status {
+    font-family: Muli, Helvetica Neue, sans-serif!important;
+    font-size: 85%;
+}
+
```

Comment:
is this needed? just curious

Reply:
It is need to make the text look just like the activate/deactivate text, I could just use the same class activate/deactivate is using instead of making another class.